### DISCUSSION TITLE
Solving unexpected token from class property inside a node_module

### DISCUSSION THREAD

**Discussion by kyle-ssg** [storybookjs/storybook#16563] [Help]
I'm currently trying to use a component within storybook that uses node module that uses class properties, I'm assuming the reason this is throwing an error is because it's not getting transpiled.

```
Unexpected token (130:8)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| export default class AnimatedBezier extends AnimatedNode {
>   _value;
|   _bezier;

```

If this is the case, is there any way to insure that a particular node_module also gets transpiled?

**Comment by franleplant** [comment]
This is so annoying, I'm also experiencing this
